api/transport/request.go

```diff
@@ -62,6 +64,19 @@ type Request struct {
 	Body io.Reader
 }
 
+// MarshalLogObject implements zap.ObjectMarshaler.
+func (r *Request) MarshalLogObject(enc zapcore.ObjectEncoder) error {
```

Comment:
Wondering about this naming convention... 

What do you think about `SetLogObject` instead of `MarshalLogObject`?

Looks like you are just setting an object *into* a `zapcore.ObjectEncoder`.